samples/reactjs-todo/client/components/journey/text-output.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * forgerock-sample-web-react
+ *
+ * textOutput.js
+ *
+ * Copyright (c) 2021 ForgeRock. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+/**
+ * @function TextOutput- React component used for displaying text Outputcallback
+ * @param {Object} props - React props object passed from parent
+ * @returns {Object} - React component object
+ */
+
+/** 
+ component added to render purposes, at this point this callback renders no element
+*/
+
+// eslint-disable-next-line no-unused-vars
+export default function TextOutput({ inputName }) {
+  return null;
```

Comment:
sorry if i missed context on this, but why are we doing this?

Reply:
some of the callbacks that webauthn steps have are of the type TextOutput and i need to handle them even if it's just rendering null, otherwise it crashes rendering the next error. I'm following the same approach i did for the angular sample app, just handling but not rendering anything.
<img width="484" alt="image" src="https://github.com/ForgeRock/forgerock-javascript-sdk/assets/18680830/d4bee315-ae19-4c4e-a46c-aba1288e5b32">
<img width="521" alt="image" src="https://github.com/ForgeRock/forgerock-javascript-sdk/assets/18680830/9947f227-bc81-40d7-90ee-3760f47b1c51">

Reply:
Ok, thank you, what is in the textoutput callback in this case?

Reply:
<img width="353" alt="image" src="https://github.com/ForgeRock/forgerock-javascript-sdk/assets/18680830/e3f4bf20-97ce-4de8-b38f-66c0cf6f30ca">

Reply:
I don't think we should `return null;` here. We should print what is present in the callbacks. Is there a reason we aren't just rendering the string that is contained in the text output callback?

Reply:
in this case, i'd print "login" or "0". Does it make sense?. we did the same fot the angular sample app

Reply:
Oh, I see what you mean. Go ahead and use the `TextOutputCallback` as it's intended. Rather than returning `null`, which just hides the message, let's render the text. 

Now, that will result in two headers essentially, "Sign In" and "Login", so let's fix that by designing a better Message Node. The message itself should ask a user a question, not act as a title, and then the options should represent the user's answer. 

Does that make sense?

BTW, if you can, avoid "resolving a conversation" when it still contains an open question or request. Thanks.